regression/goto-analyzer/pointer-differences/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--variable-sensitivity --vsd-pointers constants --vsd-values constants --verify
+^EXIT=0$
+^SIGNAL=0$
+^\[main.assertion.1\] line 16 assertion offset == 4: SUCCESS$
+^\[main.assertion.2\] line 18 assertion offset == 3: SUCCESS$
+^\[main.assertion.3\] line 20 assertion offset == 1: SUCCESS$
+^\[main.assertion.4\] line 22 assertion offset == 0: SUCCESS$
+^\[main.assertion.5\] line 26 assertion offset == 2: SUCCESS$
+^\[main.assertion.6\] line 27 assertion q == p: UNKNOWN$
```

Comment:
Another unclear one; equality should be resolvable to a difference check if nothing else.